documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1080 @@
+        namespace: <network_attachment_definition_namespace> <6>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `pod` and `multus`.
+<2> You can use either the `id` _or_ the `name` parameter to specify the source network. For `id`, specify the {osp} network UUID.
+<3> Specify a network attachment definition for each additional {virt} network.
+<4> Required only when `type` is `multus`. Specify the namespace of the {virt} network attachment definition.
+<5> Specify a network attachment definition for each additional {virt} network.
+<6> Required only when `type` is `multus`. Here, `namespace` can either be specified using the namespace property or with a name built as follows: `<network_namespace>/<network_name>`.
+endif::[]
+
+ifdef::cnv[]
+[start=3]
+. Create a `NetworkMap` manifest to map the source and destination networks:
++
+[source,yaml,subs="attributes+"]
+----
+$  cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: NetworkMap
+metadata:
+  name: <network_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        name: <network_name>
+        type: pod <1>
+      source:
+        id: <source_network_id> <2>
+        name: <source_network_name>
+    - destination:
+        name: <network_attachment_definition> <3>
+        namespace: <network_attachment_definition_namespace> <4>
+        type: multus
+      source:
+        name: <network_attachment_definition> <5>
+        namespace: <network_attachment_definition_namespace> <6>
+        type: multus
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `pod` and `multus`.
+<2> You can use either the `id` _or_ the `name` parameter to specify the source network. For `id`, specify the virtual machine.
+<3> Specify a network attachment definition for each additional {virt} network.
+<4> Required only when `type` is `multus`. Specify the namespace of the {virt} network attachment definition.
+<5> Specify a network attachment definition for each additional {virt} network.
+<6> Required only when `type` is `multus`. Here, `namespace` can either be specified using the namespace property or with a name built as follows: `<network_namespace>/<network_name>`.
+endif::[]
+
+ifdef::vmware[]
+[start=5]
+. Create a `StorageMap` manifest to map source and destination storage:
++
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: StorageMap
+metadata:
+  name: <storage_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode> <1>
+      source:
+        id: <source_datastore> <2>
+    - destination:
```

Comment:
the second mapping seems redundant (since it's exactly the same as the first one)
(also applies to the storage mappings for the other providers below)